### DISCUSSION TITLE
Access of camera (Nozzel) possible from BambuStudio ?

### DISCUSSION THREAD

**Discussion by DzzD** [OrcaSlicer/OrcaSlicer#216] [General]
Hi, I was just wondering if there is a possible external access to the Nozzle camera or Lidar from BambuStudio ? like taking a snapshot ?

Because this may be usefull in combination with a [Hough tranfrom](https://en.wikipedia.org/wiki/Hough_transform) for the pressure advance calibration on SoftFeaver. Or switching to a closed look while printing.

**Comment by Ixian** [comment]
Neither are accessible externally with one exception: Bambu Labs supports an external feed via a software-cam that can be installed from Studio, but it requires Studio to be running.

The Lidar isn't externally accessible at all. I don't even see it in the MQTT feed.
